docs/user-guide/connecting-applications.md

```diff
@@ -410,7 +410,7 @@ $ curl https://104.197.63.17:30645 -k
 Lets now recreate the Service to use a cloud load balancer, just change the `Type` of Service in the nginx-app.yaml from `NodePort` to `LoadBalancer`:
 
 ```console
-$ kubectl delete rc, svc -l app=nginx
+$ kubectl delete rc,svc -l app=nginx
```

Comment:
Otherwise you get the error `arguments in resource/name form must have a single resource and name`